apps.toml

```
@@ -542,6 +542,13 @@ state = "working"
 subtags = [ "website" ]
 url = "https://github.com/YunoHost-Apps/commento_ynh"
 
+[commet]
+category = "communication"
+potential_alternative_to = [ "Discord", "Facebook Messenger", "Signal", "Skype", "Telegram", "Whatsapp" ]
```

```suggestion
potential_alternative_to = [ "Discord", "Element", "Cinny", "Facebook Messenger", "Signal", "Skype", "Telegram", "Whatsapp" ]
```

I do not know if we should add F(L)OSS apps to that list.